Pinta.Core/Classes/Rectangle.cs

```diff
@@ -170,6 +172,8 @@ public static RectangleI Union (in RectangleI a, in RectangleI b)
 
 	public readonly RectangleI Inflated (int width, int height)
 	{
+		if (width < 0) throw new ArgumentOutOfRangeException (nameof (width));
```

Comment:
I think there are a few existing callers that pass a negative value to shrink rather than inflate, e.g. I found:
https://github.com/PintaProject/Pinta/blob/f34897cf07388854d13ca388b20034aaf85145de/Pinta.Gui.Widgets/Widgets/AnglePickerGraphic.cs#L59

Reply:
I've removed. Maybe in the future we should get an exception to be thrown if it exceeds what the rectangle can shrink?

Reply:
I'd maybe just have it return an empty rectangle (positioned at the centre of the original rectangle)?

Reply:
What would happen if the center of the rectangle is not whole number, and why?

Reply:
Yeah since these are integer coordinates I think rounding is fine and unavoidable
I mostly just meant that the result should be consistent with what you would get by repeatedly calling `Inflated (-1, -1)` until you can't shrink any farther - the result should be positioned close to the original centre